pkg/fleet/installer/default_packages.go

```diff
@@ -30,6 +30,14 @@ var defaultPackagesList = []defaultPackage{
 	{name: "datadog-agent", released: false, releasedWithRemoteUpdates: true},
 }
 
+var languageToPackage = map[string]env.ApmLibLanguage{
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 Nitpick:** ‏Should be called packageToLanguage if we keep it, no?